### PR TITLE
Add filtering information to processing docs 

### LINKED ISSUE #18: Processing Information for filtering post alevin-fry
I noticed that we have the section complete on processing with Alevin-fry, but we don't yet have information on generating the filtered single cell experiment objects, i.e. filtering with emptyDrops. We should fill in a section for how we obtained the filtered output and what parameters we used that are different from the default for emptyDrops (i.e. lower=200).

### PR DESCRIPTION
**Pull request by allyhawkins**
Closes #18. In this PR, I'm adding in the section about how we created the filtered counts matrix post processing with Alevin-fry. I added in a brief paragraph about using `DropletUtils::emptyDrops()` including a sentence about what it does, our choice of `lower=200`, and then how we actually filtered on the FDR <= 0.01. Please let me know if the explanation is at the right level of detail or not.

Another thing I wanted to note here is that we haven't yet mentioned the actual file types that we are working with or that we have both an unfiltered and filtered output. I was wondering if we should include that here somewhere or have a separate section describing what files are included for each sample (Maybe this is a good candidate for a FAQ)? 

Also in this PR I removed the headers for bulk RNA seq and spatial since that will not be part of MVP I thought it made sense to remove for now and then we can add it in when we set up the workflow for those.

### CODE REVIEW

**Comment by jaclyn-taroni** [comment]
> Another thing I wanted to note here is that we haven't yet mentioned the actual file types that we are working with or that we have both an unfiltered and filtered output. I was wondering if we should include that here somewhere or have a separate section describing what files are included for each sample (Maybe this is a good candidate for a FAQ)?

This is tracked in #6 so if you have thoughts after writing this, you can jot them down there!

**Comment by allyhawkins** [comment]
@jashapiro I changed the intro section to combine what I had with what you had suggested. Let me know what you think!

File: docs/processing_information.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ For single-nuclei samples, all counts for spliced cDNA and intronic regions were
 
 #### Filtering cells
 
-We then removed any cell barcodes that are likely to correspond to empty droplets, rather than true cells. 
+In addition to an unfiltered counts matrix, we provide a matrix filtered to only droplets that are likely to include cells, removing any cell barcodes that are likely to correspond to empty droplets.
```
Comment (jashapiro) [inline docs/processing_information.md:46]: The second clause seemed redundant to me, but including "cell barcodes" seems important, so what do you think of this?
```suggestion
In addition to an unfiltered counts matrix, we provide a matrix filtered to only cell barcodes from droplets that are likely to include true cells.
```

Comment (allyhawkins) [reply docs/processing_information.md:46]: I like it! Will go ahead and commit that change.

**Pull request merged**